### PR TITLE
Updated the Oracle Linux supported systems.

### PR DESCRIPTION
**Pull request by mmccrory**
# PR Summary

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [ ] Version 7.x preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

**Conceptual articles**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [ ] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles

## PR Checklist

- [x] I have read the [contributors guide](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/CONTRIBUTING.md) and followed the style and process guidelines
- [ ] PR has a meaningful title
- [ ] PR is targeted at the _staging_ branch
- [ ] All relevant versions updated
- [ ] Includes content related to issues and PRs - see [Closing issues using keywords](https://help.github.com/en/articles/closing-issues-using-keywords).
- [ ] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the
    title and remove the prefix when the PR is ready.

### CODE REVIEW

**Comment by sdwheeler** [comment]
@TravisEz13 Is this accurate for us?

File: reference/docs-conceptual/dsc/getting-started/lnxGettingStarted.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The following Linux operating system versions are supported for DSC for Linux.
 
 - CentOS 5, 6, and 7 (x86/x64)
 - Debian GNU/Linux 6, 7 and 8 (x86/x64)
-- Oracle Linux 5, 6 and 7 (x86/x64)
+- Oracle Linux 6.8+ and 7.8+ (x86/x64)
```
Comment (TravisEz13) [inline reference/docs-conceptual/dsc/getting-started/lnxGettingStarted.md]:
```suggestion
```
Oracle linux is not support

We should not duplicate the version list from the PowerShell LifeCycle page.

Comment (TravisEz13) [reply reference/docs-conceptual/dsc/getting-started/lnxGettingStarted.md]: In fact most of the list is wrong, or out of date.
https://docs.microsoft.com/en-us/powershell/scripting/powershell-support-lifecycle?view=powershell-7#supported-platforms

**Comment by sdwheeler** [comment]
@mmccrory Closing this PR. We are working a complete update to this article for the PS7 release. Please contact @mgreenegit and @joeyaiello regarding the updates you think need to be included.

**Pull request closed without merging**